sim/tests/execute_pytest.sh

```diff
@@ -0,0 +1,14 @@
+# !/bin/bash
```

Comment:
couldn't this be part of the Travis config?

Reply:
execute_pytest.sh is being called within the docker container called by travis, not outside of it. I thought about what was the best way to go about it, and this is the best I can think of...for now.  I can place two `docker run...`'s in script of the Travis config and remove this file, but then i guess we'll loose pytest coverage as i don't think i can pass arguments to a docker environment. 

The path should be in the dockerfile though, so i'll fix that!